packages/rx_bloc_cli/lib/src/models/command_arguments/create_command_arguments.dart

```diff
@@ -78,6 +78,15 @@ enum CreateCommandArguments {
     help: 'The deeplink showcase feature',
   ),
 
+  /// Profile feature
+  onboarding(
+    name: 'enable-feature-onboarding',
+    type: CreateCommandArgumentType.boolean,
+    defaultsTo: true,
+    prompt: 'Enabled Onboarding feature:',
+    help: 'Enables Onboarding feature capability for the project',
```

Comment:
Please add more details to the help section

Reply:
Done. Let me know if any other changes/adjustments are needed.